mantidproject/mantid
# Indirect QuickRun - add diffusion option

EnergyWindowScan has been extended to include diffraction reduction and renamed to IndirectReductionAndAnalysis to better represent what the algorithm does.

**To test:**
InputFiles: [Diff_inputs.zip](https://github.com/mantidproject/mantid/files/774174/Diff_inputs.zip)

1
Interfaces>Indirect>Diffraction
Input: `26181` 
(this should find the appropriate run in the data archive otherwise browse to `IRS26181` in files provided)
Ensure Instrument = `ISIS` and mode = `diffspec`
Press `Run`

2
```Python
IndirectReductionAndAnalysis(InputFiles='26176',
                             Instrument='IRIS',
                             Analyser='graphite',Reflection='002',
                             SpectraRange='3, 50',
                             ElasticRange='-0.5, 0',
                             InelasticRange='0, 0.5',
                             GroupingMethod='All',
                             MSDFit=False,
                             Diffraction=True,
                             DiffractionSpectraRange=[105,112])
```
Ensure irs26176_diffspec_red is the same for both methods

Fixes #18223 #12270

[Release Notes](https://github.com/mantidproject/mantid/blob/be1b1486734488150de3569407f561772a855007/docs/source/release/v3.10.0/indirect_inelastic.rst)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
